src/lang/ts/intermock.ts

```diff
@@ -122,8 +125,28 @@ function isQuestionToken(
  * @param options Intermock general options object
  */
 function processGenericPropertyType(
-    output: Output, property: string, kind: ts.SyntaxKind, mockType: string,
-    options: Options) {
+    node: ts.PropertySignature, output: Output, property: string,
+    kind: ts.SyntaxKind, mockType: string, options: Options) {
+  // @ts-ignore
+  if (node && node.type && node.type.literal &&
+      node.type.kind === ts.SyntaxKind.LiteralType) {
+    const literal = (node.type as ts.LiteralTypeNode).literal;
+    // Boolean Literal
+    if (literal.kind === ts.SyntaxKind.TrueKeyword) {
+      output[property] = true;
+    } else if (literal.kind === ts.SyntaxKind.FalseKeyword) {
+      output[property] = false;
+      // String Literal
+    } else if (literal.kind === ts.SyntaxKind.StringLiteral) {
+      output[property] = literal.text ? literal.text : '';
+      // Numeric Literal
+    } else {
+      // The text IS a string, but the output value has to be a numeric value
+      // tslint:disable-next-line
```

Comment:
I'm not a big fan of turning of the linter unless it's absolutely necessary. Usually it means there's some API problem somewhere. How about making `Output` a container for strings as well as numbers?

Reply:
I'm not a fan either. I was debugging the NumericLiteral object and I couldn't find the actual value that it requires as a number, I could only find it as a string, and it being a number literal I needed to parse it. If you have another solution in mind, I'd be glad to use it

Reply:
I think if you allow the `Record` type of `Output` to be `string|number` this could fix it? https://github.com/google/intermock/blob/master/src/lang/ts/intermock.ts#L66

Reply:
Tried it, but the I can't find the actual numerical value that the `NumericLiteral` requires on the object. Here's a screenshot of the object:

<img width="621" alt="Screen Shot 2019-07-30 at 10 20 21 AM" src="https://user-images.githubusercontent.com/41019696/62137462-fc1a8f80-b2b3-11e9-9756-c6ab4cbda4b8.png">

As you can see on the image, the only place where the 5 appears is on the text property